assignments/config-files/hackyoutemperature/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "hackyoutemperature",
+  "version": "1.0.0",
+  "type": "module",
+  "main": "app.js",
+  "scripts": {
+    "test": "jest"
+  },
```

It would be nice to have more scripts here such as:

```suggestion
  "scripts": {
    "start": "node server.js",
    "dev": "nodemon server.js",
    "test": "jest"
  },
```